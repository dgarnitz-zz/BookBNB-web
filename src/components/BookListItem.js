import React from 'react';
import TableCell from '@material-ui/core/TableCell';
import TableRow from '@material-ui/core/TableRow';

function BookListItem(props){
  let availableColumn = null;
  if(props.available) {
    availableColumn = <TableCell>{props.status}</TableCell>;
  }

  return (
    <TableRow>
      <TableCell>{props.title}</TableCell>
      <TableCell>{props.author}</TableCell>
      <TableCell>1st edition</TableCell>
      {availableColumn}
    </TableRow>
  );
}

export default BookListItem;
