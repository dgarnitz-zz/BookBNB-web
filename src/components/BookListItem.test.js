import React from "react";
import BookListItem from "./BookListItem";
import renderer from "react-test-renderer";

test("test_one", () => {
  const component = renderer.create(<BookListItem />);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
