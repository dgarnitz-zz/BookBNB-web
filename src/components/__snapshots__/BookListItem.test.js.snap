// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test_one 1`] = `
<tr
  className="MuiTableRow-root-1"
>
  <td
    aria-sort={null}
    className="MuiTableCell-root-6"
  />
  <td
    aria-sort={null}
    className="MuiTableCell-root-6"
  />
  <td
    aria-sort={null}
    className="MuiTableCell-root-6"
  />
</tr>
`;
